packages/arb-token-bridge-ui/src/components/TransferPanel/TokenSearch.tsx

```diff
@@ -271,8 +292,10 @@ function TokenRow({ style, onClick, token }: TokenRowProps): JSX.Element {
             {isArbitrumToken && (
               <Tooltip content={arbitrumTokenTooltipContent}>
                 <StatusBadge variant="green">
-                  <CheckCircleIcon className="h-4 w-4" />
-                  <span className="text-xs">Official ARB token</span>
+                  {!isSmallScreen && <CheckCircleIcon className="h-4 w-4" />}
+                  <span className="text-xs">
+                    Official ARB{isSmallScreen ? '' : ' token'}
```

Comment:
for mobile, the following copy looks better:
`✔ Official`

I'd keep the checkmark and remove the other words

Reply:
updated